### PR TITLE
Remove details summary webkit styles

### PR DESCRIPTION
**Pull request by dumindu**
Safari 
Screenshot 2023-11-06 at 5 24 40 PM

### CODE REVIEW

File: css/reset.css [hunk 1]
```diff
@@ -75,9 +75,10 @@ meter {
     color: unset;
 }
 
-/* fix the feature of 'hidden' attribute.
-   display:revert; revert to element instead of attribute */
-:where([hidden]) {
+/* - Fix the feature of 'hidden' attribute.
+    - display:revert; revert to element instead of attribute
+   - Remove details summary webkit styles */
+:where([hidden]), details summary::-webkit-details-marker {
```
Comment (elektronik2k5) [inline css/reset.css:81]: Can we do this with lower specificity?
```suggestion
:where([hidden]), summary::-webkit-details-marker {
```

**Comment by fabulousgk** [comment]
IMO this should be seperated out as its own selector, as it has nothing to do with display-hidden other than sharing the same property.

Would be clearer I belive.

Also only the psuedo is needed.

::-webkit-details-marker {
  display: none;
}

**Comment by fabulousgk** [comment]
Alos stupid Safari, I mean for reals....lol

**Comment by dumindu** [comment]
Can use directly `::-webkit-details-marker` but should I group all similar style resets into 1 block or not?

 Ex. Both `:where([hidden])` and `::-webkit-details-marker` having only `display: none`

cc @elad2412 @elektronik2k5 @fabulousgk

**Comment by fabulousgk** [comment]
Either way works, I just personally like it separated as to me it is more clear there are two separate issues being addressed.  That is just an opinion though.

I can see the arguments for them being together as well.

**Comment by fabulousgk** [comment]
Look at dialog and pre as examples of existing items kept separate.

**Comment by dumindu** [comment]
Make sense. Updated.

**Pull request merged**